server/services/bigtable.py

```diff
@@ -48,19 +58,36 @@ def get_project_id():
   return project_id
 
 
-async def write_row(session_info):
+async def write_feedback(session_info, data):
+  project_id = get_project_id()
+  row_key = get_row_key(session_info, project_id)
+  table = current_app.config['NL_TABLE']
+  row = table.direct_row(row_key)
+  row.set_cell(_COLUMN_FAMILY, _COL_FEEDBACK.encode(), json.dumps(data))
```

Comment:
Since we can get multiple feedbacks for the same session_id, and even query, is the caller expected to send the full "data" covering all prior feedbacks?  If not should we try to append or something?

Reply:
Yes. I am not fully sure how the data looks. Will update this placeholder with the client side change.